src/components/Releases/ReleaseDetailsView.tsx

```diff
@@ -63,6 +71,31 @@ const ReleaseDetailsView: React.FC = () => {
         applicationName,
         releaseName,
       })}
+      actions={[
+        {
+          onClick: () => {
+            track(TrackEvents.ButtonClicked, {
+              link_name: 're-run-release',
+              link_location: 'release-actions',
+              release_name: releaseName,
+              app_name: applicationName,
+              namespace,
+            });
+            void releaseRerun(release, email).then(() => {
+              navigate(
+                APPLICATION_RELEASE_LIST_PATH.createPath({
+                  workspaceName: namespace,
+                  applicationName,
+                }),
+              );
+            });
+          },
+          disabledTooltip: 'You do not have access to re-run release',
+          isDisabled: !release,
```

Comment:
here you need to check whether create permission is there, 
`const canCreateRelease = useAccessReviewForModel(ReleaseModel, 'create')`

`isDisabled: !canCreateRelease`